### PR TITLE
Add disabled state to button

### PR DESCRIPTION
**Pull request by jreis13**
Why:
* https://3.basecamp.com/4319812/buckets/26581088/todos/5215871363
* Design requires us to have a disabled button on the privacy policy page

How:
* Adding css rules so that we can display the button differently when its disabled

### CODE REVIEW

File: frontend/src/components/Button/index.module.css [hunk 1]
```diff
@@ -15,3 +15,8 @@
   border-radius: 300px;
   box-shadow: 0 4px 3px var(--color-gray);
 }
+
+.button:disabled {
+  color: var(--color-dark-gray);
+  background-color: var(--color-light-gray);
+}
```
Comment (mirhamasala) [inline frontend/src/components/Button/index.module.css:22]: Why not use `sass` now that we have it installed?

```suggestion
  box-shadow: 0 4px 3px var(--color-gray);
  
  &:disabled {
    color: var(--color-dark-gray);
    background-color: var(--color-light-gray);
   }
}
```

Comment (mirhamasala) [reply frontend/src/components/Button/index.module.css:22]: P.S. Not sure if you've tried this already, but if you agree, you can commit my suggestion straight from here. Look for the `Commit suggestion` button

**Review by mirhamasala — APPROVED** [review]
Minor suggestion to use SASS. ✨🚀

File: frontend/src/components/Button/index.module.css (hunk 1 above)
Comment (jreis13) [reply frontend/src/components/Button/index.module.css:22]: That's a good point, I'll do that 🙏

**Pull request merged**